docs/quick-start/index.md

```diff
@@ -896,16 +960,16 @@ of a solution based on Spine Event Engine framework.
 The development starts with the discovery of the business using EventStorming or another
 learning approach.
 
-Then, we select a Bounded Context and define events, commands, Value Objects and states of entities
+Then, we select a Bounded Context and define events, commands, Value Objects and entity states
 using Protobuf. Using these `.proto` files Spine Model Compiler generates the code implementing
 the defined data types.
 
 After that we add the business logic for handling commands or events in entity classes derived from
 `Aggregate`, `ProcessManager`, or `Projection`. 
 
-Then, the selected Bounded Context is assembled using these entity types and tested using
-the black-box approach. A test suite sends signals (e.g. commands or events) to the implementation
-of the Bounded Context and verifies the changes manifested as events or new states of the entities.
+Then,  these entity types are assembled into a Bounded Context and tested as a whole.
+A test suite sends signals (e.g. commands or events) to the implementation of the Bounded Context
```

Comment:
Should it be `i.e.` instead of `e.g.`? Do we have some other signals that are not commands or events?
```suggestion
A test suite sends signals (i.e. commands or events) to the implementation of the Bounded Context
```